source/cloud/aws/sagemaker.md

```diff
@@ -104,33 +125,12 @@ hpo = sagemaker.tuner.HyperparameterTuner(
     max_jobs,
     max_parallel_jobs,
 )
-hpo.fit(inputs=s3_data_input, job_name=tuning_job_name, wait=True, logs="All")
-```
-
-### Upload data to S3
-
-We offer the dataset for this demo in a public bucket hosted in either the [`us-east-1`](https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-east-1/) or [`us-west-2`](https://s3.console.aws.amazon.com/s3/buckets/sagemaker-rapids-hpo-us-west-2/) regions
```

Comment:
This was just incorrect. Those buckets exist and data exists there, but not any datasets references in this document or the things it links to.

We can safely remove this... the Higgs notebook linked to in this doc handles downloading a dataset and putting it up on S3 itself.

https://github.com/rapidsai/deployment/blob/3578ead5358699a94b6668992b21578f8e81fc05/source/examples/rapids-sagemaker-higgs/notebook.ipynb?short_path=401b666#L92-L106